### DISCUSSION TITLE
Why not cppan?

### DISCUSSION THREAD

**Discussion by Telokis** [microsoft/vcpkg#20699] [Q&A]
In the [faq](https://github.com/Microsoft/vcpkg/blob/master/docs/about/faq.md) you forgot to consider another solution which, in my opinion, deserves to be considered: [cppan](https://github.com/cppan/cppan).
On a feature-based level, it seems to be more powerful than vcpkg so I don't understand why you didn't mention it:
- It is cross-platform so there is only a single dependency file to consider.
- It supports multiple versions of the same library, in case you have two projects depending on two different versions
- I don't think I need to mention anything else. Those two points seem to be enough for me.

**Comment by linquize** [comment]
**cppan** relies on its own server to fetch the library definitions. Every time you wish to improve a library, you have to submit requests to cppan server. Also, cppan long term commitment, and reliablity are not guaranteed.

**vcpkg** library definitions are in one git repository, and git the repository can be cloned for offline use. The repository is hosted on github so it is more reliable.
You can customize vcpkg repository too.

**Comment by linquize** [comment]
**cppan** v1 is obsolete, now it is migrated to **sw**.

**Comment by egorpugin** [comment]
Yes, this should be renamed to `Why not SW?`. SW is cppan v.2.
For reference: https://github.com/SoftwareNetwork/sw
